en/docs/api-analytics/samples/publishing-analytics-events-to-external-systems.md

```diff
@@ -1,250 +1,276 @@
-
-This section will cover the steps required to create a sample, configure the created sample with WSO2 API-M or Choreo Connect, and then publish it to an external System (ELK).
-
-## Creating the Sample
-
-You have to create a new `Java/Maven project`. 
+WSO2 API Manager allows publishing its analytics data to external systems in the same way it publishes the data to the cloud. For this purpose you need to create a custom event publisher. This guide will explain and walk through the steps required to implement, deploy, and configure a custom event publisher.
 
-With the use of `WSO2 Analytics Publisher` extension which is available at `WSO2 nexus` repository it is possible to create the sample. Created sample can be used to log and publish analytics events to external systems.
+For demonstration purposes, let’s assume ELK as the external system.
 
-There is an already [created sample](https://github.com/wso2/samples-apim/tree/master/analytics-event-publisher) and if you wish to use that sample instead of developing the sample from scratch, then you can ignore the steps of creating the sample and start from [here]({{base_path}}/api-analytics/samples/publishing-analytics-events-to-external-systems/#build-the-project).
+## Step 1 - Create an Event Publisher
 
-This section will cover how to configure the `pom.xml`, class implementations and building the created sample.
+Follow the instructions below to create the custom event publisher.
 
-### Configuring pom.xml
+!!! Note
+    If you are not interested in creating the custom event publisher from scratch, you can use the already created [sample event publisher](https://github.com/wso2/samples-apim/tree/master/analytics-event-publisher) instead and resume the guide from here.
 
-Add wso2-nexus repository to `pom.xml`,
+### Step 1.1 - Set up a Maven project
 
-    <repository>
-        <id>wso2-nexus</id>
-        <name>WSO2 internal Repository</name>
-        <url>https://maven.wso2.org/nexus/content/groups/wso2-public/</url>
-        <releases>
-            <enabled>true</enabled>
-            <updatePolicy>daily</updatePolicy>
-            <checksumPolicy>ignore</checksumPolicy>
-        </releases>
-    </repository>
+1. Create a new Maven-based Java project.
 
-Add dependency,
+2. Add the required dependency.
+     
+     The Maven-based Java project uses the [WSO2 Analytics Publisher](https://maven.wso2.org/nexus/content/repositories/releases/org/wso2/am/analytics/publisher/) library, which is available in the WSO2 Nexus repository. Therefore, you need to add this library as a dependency.
 
-    <dependency>
-        <groupId>org.wso2.am.analytics.publisher</groupId>
-        <artifactId>org.wso2.am.analytics.publisher.client</artifactId>
-        <version>1.0.1</version>
-    </dependency>
+     1. Navigate and open the `pom.xml` file.
```

Comment:
@lasanthaS , shall we mention the directory path of the `pom.xml` file.

Reply:
Let's rename it as `<PROJECT_DIR>/pom.xml`